src/components/RegionInput.js

```diff
@@ -6,7 +6,6 @@ import FormHelperText from "@mui/material/FormHelperText";
 
 // RegionInput: The path and region input box component
 // Responsible for selecting the path/chr and segment of data to look at
-const isEmpty = (obj) => Object.keys(obj).length === 0;
```

Comment:
I think you need to keep this in for the component to work.

Reply:
Oh I guess we want to import the new `isEmpty` function from `common.mjs`. Which we can do if we import `config-client.js` first.